src/js/domManipulation.js

```diff
@@ -0,0 +1,186 @@
+    shufflePlaylist.forEach((data) => {
+        const recentPlaylistItem = document.createElement("li");
+        recentPlaylistItem.className = "recent-playlist-item";
+        recentPlaylistItem.innerHTML = `
+            <img class="recent-playlist-img" src="${data.artistImage}" alt="artist">
+            <div class="recent-playlist-details">
+                <h3 class="fs-18 fw-500">${data.title}</h3>
+                <span class="block fs-14 fw-500">${data.artist}</span>
+                <small class="fs-14">${data.studio} . ${data.year}</small>
+            </div>
+            <div class="recent-playlist-controls">
+                <small>${data.duration}</small>
+                <span class="recent-playlist-play-btn">
+                    <i class="fa-solid fa-play recent-playlist-play-icon text-primary-color"></i>
+                </span>
+            </div>
+        `
+        recentPlayList.appendChild(recentPlaylistItem);
+    })
+}
+
+const createArtist = dataList => {
+    dataList.forEach(data => {
+        const artist = document.createElement('li');
+        artist.className = "artist-item";
+        artist.innerHTML = `
+            <img class="artist-img" src="${data.artistImage}" alt="artist">
+            <span class="artist-name">${data.artist}</span>
+        `;
+        artistList.appendChild(artist);
+    })
+}
+
+const favoriteCard = (dataList) => {
+    dataList.forEach((data, index) => {
+        const artistCardItem = createArtistCard(data, index);
+        favoriteList.appendChild(artistCardItem);
+    })
+}
+
+
+const shuffleData = (data) => {
+    const shuffledArr = data.slice();
+    for(let i= shuffledArr.length - 1; i > 0; i--){
+        const randomIndex = Math.floor(Math.random() * (i + 1));
+        [shuffledArr[i], shuffledArr[randomIndex]] = [shuffledArr[randomIndex], shuffledArr[i]]
+    }
+    return shuffledArr;
+}
+
+
+export const getDataUrl = (e) => {
+    let file = e.target.files[0];
+    let fileReader = new FileReader();
+    fileReader.onload = (e) => {
+        previewProfilePic.src = e.target.result;
+    }
+    fileReader.readAsDataURL(file);
+    console.log("saveProfile function called");
+
+}
+
+export const displayUserProfile = (data) => {
+    profileNameElement.textContent = data.name;
+    displayUsername.textContent = `Hello ${data.name}`;
+    if (data.image) {
+        profilePictureElement.src = data.image;
+    }
+}
+export const saveProfile = () =>{
+    if(profileNameInput.value !== "" ){
+        profileData.name = profileNameInput.value;
+    }
+    profileData.image = previewProfilePic.src;
+    localStorage.setItem('uploadProfile', JSON.stringify(profileData));
+    profileNameInput.value = "";
+    displayUserProfile(profileData);
+
+}
+
+export const PreviewProfile = () => {
+    profileModal.style.display = "block";
+    previewProfilePic.src = profileData.image;
+}
+
+export const cancelPreviewProfile = () => {
+    profileModal.style.display = "none";
```

Comment:
since you are using `display: none;` in multiple places - maybe this [comment](https://github.com/Dina-petrick/sound-buzz/pull/5/files#r1270495727) will make it easier